src/math/bspline.hpp

```diff
@@ -11,6 +11,16 @@ namespace precice::math::bspline {
  * @pre splineDegree must be larger than 0.
  * @pre t must be within [ts.min; ts.max].
  */
-Eigen::VectorXd interpolateAt(Eigen::VectorXd ts, const Eigen::MatrixXd &xs, int splineDegree, double t);
+class bspline {
 
+public:
+  Eigen::VectorXd interpolateAt(Eigen::VectorXd ts, const Eigen::MatrixXd &xs, int splineDegree, double t, bool recomputeBuffer);
+
+private:
+  //interpolate(xs, splineDegree, ts, tRelative, ndofs);
+  Eigen::VectorXd interpolate(const Eigen::MatrixXd pts, int degree, Eigen::VectorXd knot_parameters, double tRelative, int ndofs);
+  void            ComputeMatrices(const Eigen::MatrixXd pts, int degree, Eigen::VectorXd knot_parameters, int ndofs);
```

Comment:
```suggestion
  void            computeMatrices(const Eigen::MatrixXd pts, int degree, Eigen::VectorXd knot_parameters, int ndofs);
```
Because it is a function.